tests/server_test_cases/config_tests.py

```diff
@@ -440,7 +440,7 @@ def test_analytics_id(self):
         config.set(analytics_id='analytics_id_xyz')
 
         doc = self.go('/haiti/create')
-        assert "getTracker('analytics_id_xyz')" in doc.content
+        assert "gaProperty': 'analytics_id_xyz'" in doc.content
```

Comment:
Minor comment, but it would be more natural to include the first quotation mark too in the assert string: 'gaProperty': 'analytics_id_xyz'